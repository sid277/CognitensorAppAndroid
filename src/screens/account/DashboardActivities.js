import React from 'react';
import { Text } from 'react-native';
import DefaultView from '../../components/default/DefaultView';

const DashboardActivities = () => {
  return (
    <DefaultView>
      <Text>Dashboard Activities</Text>
    </DefaultView>
  );
};

export default DashboardActivities;
