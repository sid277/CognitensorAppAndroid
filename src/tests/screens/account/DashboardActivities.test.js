import React from 'react';
import { shallow } from 'enzyme';
import DashboardActivities from '../../../screens/account/DashboardActivities';
import renderer from 'react-test-renderer';


test('should test DashboardActivities component', () => {
const wrapper = shallow(<DashboardActivities />);
expect(wrapper).toMatchSnapshot();
});