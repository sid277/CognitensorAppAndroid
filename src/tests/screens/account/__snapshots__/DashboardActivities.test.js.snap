// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test DashboardActivities component 1`] = `ShallowWrapper {}`;
